openstudiocore/src/model/AirLoopHVAC.cpp

```diff
@@ -914,6 +914,25 @@ namespace detail {
     bool result = setString(OS_AirLoopHVACFields::DesignSupplyAirFlowRate, "autosize");
     OS_ASSERT(result);
   }
+  
+  /** Baseline Pressure Drop Adjustment **/
+  // Return the Baseline Pdrop
+  double AirLoopHVAC_Impl::baselinePressureDropAdjustment() const {
+    return getDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment,true);
```

Comment:
Does this compile?  I would expect you to have to do this:

    boost::optional<double> result = getDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment,true);
    OS_ASSERT(result); // ok to assert since there is a default in the IDD, want to assert because .get below will cause a crash if result is empty
    return result.get();